examples/run_pulse_calibration.py

```diff
@@ -0,0 +1,193 @@
+r"""
+Use the Scope to Calibrate pulses output from RF amplifier
+===========================================================
+If calibrating the pulse lengths, a series of pulse lengths (μs) are directly
+output from the SpinCore to the rf amplifier where the output pulse is captured
+on the GDS oscilloscope.  
+If testing the calibration or capturing using a series of desired betas, the
+calibrating conditional should be set to False and the script will calibrate
+the pulse lengths based on the amplitude set in the active.ini file so that the
+output of the amplifier produces the desired beta.  
+
+Note
+----
+    There MUST BE at least a 40 dBm attenuator between the RF amplifier output
+    and the GDS oscilloscope input to avoid damaging the instrumentation! It is
+    advised that the attenuator be calibrated using the GDS and AFG beforehand.
+"""
+
+import pyspecdata as psd
+import os
+import time
+import SpinCore_pp as spc
+from datetime import datetime
+from Instruments import GDS_scope
+from numpy import r_
+import numpy as np
+
+calibrating = True
+
+nominal_power = 75  # in W
+nominal_atten = 1e4
+num_div_per_screen = 8
+n_lengths = 50  # number of pulse lengths acquired
+indirect = "t_pulse" if calibrating else "beta"
+my_exp_type = "test_equipment"
+assert os.path.exists(psd.getDATADIR(exp_type=my_exp_type))
+tx_phases = np.r_[0.0, 90.0, 180.0, 270.0]
+# {{{ importing acquisition parameters
+config_dict = spc.configuration("active.ini")
+(
+    nPoints,
+    config_dict["SW_kHz"],
+    config_dict["acq_time_ms"],
+) = spc.get_integer_sampling_intervals(
+    config_dict["SW_kHz"], config_dict["acq_time_ms"]
+)
+# }}}
+# {{{ add file saving parameters to config dict
+config_dict["type"] = "pulse_calib"
+config_dict["date"] = datetime.now().strftime("%y%m%d")
+config_dict["pulse_calib"] += 1
```

Comment:
This is not called "counter"... Might be my bad. 

Also, are the three lines here not handled by the file saving function? In particular, the date?